UPGRADING.md

```diff
@@ -1,6 +1,20 @@
 Upgrading iex-ruby-client
 =========================
 
+### Upgrading to >= 2.0.0
+[#113](https://github.com/dblock/iex-ruby-client/pull/113) Explicitly updates the minimum Ruby version to 2.4* from 2.3 and removes default values for Faraday's SSL settings `ca_file` and `ca_path`.
+
+If you previously relied on `OpenSSL::X509::DEFAULT_CERT_FILE` or `OpenSSL::X509::DEFAULT_CERT_DIR` to set these values you must now do so explicitly. E.g.:
+
+```ruby
+IEX::Api.configure do |config|
+  config.ca_file = OpenSSL::X509::DEFAULT_CERT_FILE
+  config.ca_path = OpenSSL::X509::DEFAULT_CERT_DIR
+end
+```
+
+\*At this time Ruby 2.7 is the earliest non-EOL branch. To maintain maximum compatibility 2.4 was chosen as `transform_values` was already being used and first appears in 2.4.
```

Comment:
Make two subsections under upgrading to >= 2.0, one for SSL cert options and another for Ruby minimum. Just say that the minimum ruby version is X, no need for a long explanation.